Packs/Exabeam/Integrations/Exabeam/Exabeam.py

```diff
@@ -1895,6 +1947,66 @@ def get_notable_sequence_event_types(client: Client, args: Dict[str, str]) -> Tu
     return human_readable, entry_context, sequence_event_types_raw_data
 
 
+def fetch_ir_as_incidents(client: Client, args: Dict[str, str]):
```

Comment:
please call it in the rest module in case that fetch is enabled.

Reply:
fixed